### PR TITLE
Adding comment on define_enum_for_matcher.rb

### PR DESCRIPTION
**Pull request by Kutomore**
Fix for #927

### CODE REVIEW

File: lib/shoulda/matchers/active_record/define_enum_for_matcher.rb [hunk 1]
```diff
@@ -43,6 +43,34 @@ module ActiveRecord
       #         with_values([:running, :stopped, :suspended])
       #     end
       #
+      # If you specify your enum with numbers that don't follow the
```
[the remark this answers is not in the document]
Comment (Kutomore) [reply lib/shoulda/matchers/active_record/define_enum_for_matcher.rb]: You're right, it does sound better and more professional. I'll change it right now, I guess I just couldn't think about a better alternative :p

File: lib/shoulda/matchers/active_record/define_enum_for_matcher.rb [hunk 2]
```diff
@@ -43,6 +43,35 @@ module ActiveRecord
       #         with_values([:running, :stopped, :suspended])
       #     end
       #
+      # If the values backing your enum attribute are arbitrary instead of a
+      # series of integers starting from 0, pass a hash to with_values instead
+      # of an array:
+      #
+      #     class Process < ActiveRecord::Base
+      #         enum status: {
```
[the remark this answers is not in the document]
Comment (Kutomore) [reply lib/shoulda/matchers/active_record/define_enum_for_matcher.rb]: Just to make it clear, you man changing it from 4 spaces to 2 spaces?

[the remark this answers is not in the document]
Comment (mcmire) [reply lib/shoulda/matchers/active_record/define_enum_for_matcher.rb]: Yup!

**Comment by Kutomore** [comment]
@mcmire Hi, so the CI Build seems to have failed, but the only thing I changed on this PR are comments, can you rerun it please?

**Comment by mcmire** [comment]
Yeah, hmm, we have some flakiness issues right now it seems. I've restarted the build.

**Comment by mcmire** [comment]
Sorry it took me so long to get to this. Thanks for the PR! I've merged it as c47e6db7391963d27cb9b45c37d9aa94e53f0590.

**Pull request closed without merging**